### PR TITLE
feat : creat dash/userinfo api and add autherisation middleware to th…

### PR DESCRIPTION
**Pull request by jayesh315thinkitive**
…is api

### CODE REVIEW

File: basic-rest-apis/rest-api-project/controllers/auth.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const { validationResult } = require("express-validator/check");
 const bcrypt = require("bcryptjs");
 const User = require("../models/user");
 const jwt = require("jsonwebtoken");
-require('dotenv').config()
+require("dotenv").config();
 
 exports.signUp = (req, res, next) => {
   const errors = validationResult(req);
```
Comment (imtiyazthink) [inline basic-rest-apis/rest-api-project/controllers/auth.js:8]: Add try catch blocks for better exception handling

File: basic-rest-apis/rest-api-project/controllers/dashboard.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+const express = require('express')
+const User = require("../models/user")
+
+
+exports.userInfo = (req,res,next) =>{
+
+ User.find()
```
Comment (imtiyazthink) [inline basic-rest-apis/rest-api-project/controllers/dashboard.js:8]: Add try catch blocks for better exception handling

File: basic-rest-apis/rest-api-project/controllers/feed.js [hunk 3]
```diff
@@ -3,7 +3,6 @@ const Post = require("../models/post");
 const fs = require("fs");
 const path = require("path");
 const User = require("../models/user");
-const user = require("../models/user");
 
 exports.getPosts = (req, res, next) => {
   Post.find()
```
Comment (imtiyazthink) [inline basic-rest-apis/rest-api-project/controllers/feed.js:8]: Add try catch blocks for better exception handling

**Review by imtiyazthink — CHANGES_REQUESTED** [review]

File: basic-rest-apis/rest-api-project/controllers/dashboard.js (hunk 2 above)
Comment (jayesh315thinkitive) [reply basic-rest-apis/rest-api-project/controllers/dashboard.js]: updated

**Pull request merged**